pkg/destroy/openstack/swift.go

```diff
@@ -0,0 +1,43 @@
+package openstack
+
+import (
+	"github.com/gophercloud/gophercloud/openstack/objectstorage/v1/containers"
+	"github.com/gophercloud/gophercloud/openstack/objectstorage/v1/objects"
+	"github.com/gophercloud/utils/openstack/clientconfig"
+)
+
+// DeleteSwiftContainer deletes a container and all of its objects.
+func DeleteSwiftContainer(name string, cloud string) error {
```

Comment:
It would be nice if we can move this function to `pkg/destroy/bootstrap/bootstrap.go` and use it there directly, but I don't mind to keep it here.

Reply:
I originally put it in a new directory `pkg/destroy/bootstrap/openstack/swift.go` it seemed redundant. I wasnt sure if @abhinavdahiya would take issue with openstack functions being in `pkg/destroy/bootstrap/bootstrap.go`. @abhinavdahiya do you have a preference?